diff --git a/ConfigurationValidation.hpp b/ConfigurationValidation.hpp
index 74667a1..82ed67e 100644
--- a/ConfigurationValidation.hpp
+++ b/ConfigurationValidation.hpp
@@ -90,8 +90,7 @@
         #endif
     #endif
 #elif defined(BOARD_OAE_V1)
-    // Valid 
-
+// Valid
 
 #else
     #error Configuration does not support AZ. Use at own risk.
@@ -114,7 +113,7 @@
     #endif
 
 #elif defined(BOARD_OAE_V1)
-    // Valid 
+// Valid
 
 #else
     #warning Configuration does not support ALT. Use at own risk.
diff --git a/Configuration_adv.hpp b/Configuration_adv.hpp
index 09ccb3e..cd16833 100644
--- a/Configuration_adv.hpp
+++ b/Configuration_adv.hpp
@@ -340,13 +340,12 @@
             #ifndef AZ_ROD_PITCH
                 #define AZ_ROD_PITCH 0.5
             #endif
-            #define AZIMUTH_STEPS_PER_REV   +                                                                                                  \
-                (AZ_CORRECTION_FACTOR * (AZ_CIRCUMFERENCE / AZ_ROD_PITCH) * AZ_STEPPER_SPR * AZ_MICROSTEPPING)  // Actually u-steps/rev
+            #define AZIMUTH_STEPS_PER_REV                                                                                                  \
+                +(AZ_CORRECTION_FACTOR * (AZ_CIRCUMFERENCE / AZ_ROD_PITCH) * AZ_STEPPER_SPR * AZ_MICROSTEPPING)  // Actually u-steps/rev
         #else
             #define AZ_CIRCUMFERENCE 2538.4f
         #endif
     #endif
-    
 
     #ifndef OAE
         #define AZ_WORMGEAR_RATIO 1.0f
@@ -357,7 +356,7 @@
             (AZ_CORRECTION_FACTOR * (AZ_CIRCUMFERENCE / (AZ_PULLEY_TEETH * GT2_BELT_PITCH)) * AZ_STEPPER_SPR                               \
              * AZ_MICROSTEPPING)  // Actually u-steps/rev
     #endif
-    
+
     #ifndef AZIMUTH_STEPS_PER_ARC_MINUTE
         #define AZIMUTH_STEPS_PER_ARC_MINUTE (AZIMUTH_STEPS_PER_REV / (360 * 60.0f))  // Used to determine move distance in steps
     #endif
@@ -416,15 +415,16 @@
         #define ALTITUDE_STEPS_PER_REV                                                                                                     \
             (ALT_CORRECTION_FACTOR * (ALT_CIRCUMFERENCE / ALT_ROD_PITCH) * ALT_STEPPER_SPR * ALT_MICROSTEPPING)  // Actually u-steps/rev
 
-    #else 
+    #else
         #ifdef OAE
             #ifndef ALT_ROD_PITCH
                 #define ALT_ROD_PITCH 1.25  // mm/rev
             #endif
             // the Circumference of the ALT rotation. Roughly 146mm radius.
             #define ALT_CIRCUMFERENCE 130.0f * 2 * PI
-            #define ALTITUDE_STEPS_PER_REV   +                                                                                                  \
-                (ALT_CORRECTION_FACTOR * (ALT_CIRCUMFERENCE / ALT_ROD_PITCH) * ALT_STEPPER_SPR * ALT_MICROSTEPPING)  // Actually u-steps/rev
+            #define ALTITUDE_STEPS_PER_REV                                                                                                 \
+                +(ALT_CORRECTION_FACTOR * (ALT_CIRCUMFERENCE / ALT_ROD_PITCH) * ALT_STEPPER_SPR                                            \
+                  * ALT_MICROSTEPPING)  // Actually u-steps/rev
         #else
             // the Circumference of the AZ rotation. 770mm dia.
             #define ALT_CIRCUMFERENCE 2419.0f
@@ -436,9 +436,9 @@
                 #define ALT_WORMGEAR_RATIO (40.0f)
             #endif
             #ifndef ALTITUDE_STEPS_PER_REV
-                #define ALTITUDE_STEPS_PER_REV                                                                                                 \
-                    (ALT_CORRECTION_FACTOR * (ALT_CIRCUMFERENCE / (ALT_PULLEY_TEETH * GT2_BELT_PITCH)) * ALT_STEPPER_SPR * ALT_MICROSTEPPING   \
-                    * ALT_WORMGEAR_RATIO)  // Actually u-steps/rev
+                #define ALTITUDE_STEPS_PER_REV                                                                                             \
+                    (ALT_CORRECTION_FACTOR * (ALT_CIRCUMFERENCE / (ALT_PULLEY_TEETH * GT2_BELT_PITCH)) * ALT_STEPPER_SPR                   \
+                     * ALT_MICROSTEPPING * ALT_WORMGEAR_RATIO)  // Actually u-steps/rev
             #endif
         #endif
     #endif
@@ -649,7 +649,6 @@
 //                                  ////////
 ////////////////////////////////////////////
 
-
 // Focuser
 #if (FOCUS_DRIVER_TYPE == DRIVER_TYPE_TMC2209_UART)
     #if defined(ESP32)
diff --git a/Constants.hpp b/Constants.hpp
index 17c0042..f4a9ddd 100644
--- a/Constants.hpp
+++ b/Constants.hpp
@@ -18,7 +18,7 @@
 
 // ESP32 based boards
 #define BOARD_ESP32_ESP32DEV 1001
-#define BOARD_OAE_V1 1002
+#define BOARD_OAE_V1         1002
 
 /**
  * Supported keypad/display types. Use one of these values for DISPLAY_TYPE configuration matching your used display and keypad.
diff --git a/boards/ESP32_ESP32DEV/pins_OAE_V1.hpp b/boards/ESP32_ESP32DEV/pins_OAE_V1.hpp
index 1bdb4b9..00d3db6 100644
--- a/boards/ESP32_ESP32DEV/pins_OAE_V1.hpp
+++ b/boards/ESP32_ESP32DEV/pins_OAE_V1.hpp
@@ -83,13 +83,13 @@
 
 // DISPLAY_TYPE_LCD_JOY_I2C_SSD1306 requires 3 analog inputs in Arduino pin numbering
 #ifndef LCD_KEY_SENSE_X_PIN
-    //#define LCD_KEY_SENSE_X_PIN 34
+//#define LCD_KEY_SENSE_X_PIN 34
 #endif
 #ifndef LCD_KEY_SENSE_Y_PIN
-    //#define LCD_KEY_SENSE_Y_PIN 39
+//#define LCD_KEY_SENSE_Y_PIN 39
 #endif
 #ifndef LCD_KEY_SENSE_PUSH_PIN
-    //#define LCD_KEY_SENSE_PUSH_PIN 36
+//#define LCD_KEY_SENSE_PUSH_PIN 36
 #endif
 
 //Serial port for external debugging
@@ -103,7 +103,6 @@
     #endif
 #endif
 
-
 // Defines for OAE ///////////////////////
 
 #ifndef RA_WHEEL_CIRCUMFERENCE
@@ -125,7 +124,7 @@
     #define DEC_WHEEL_CIRCUMFERENCE 1.0f
 #endif
 #ifndef RA_STEPPER_SPR
-    #define RA_STEPPER_SPR (400 * 9)   // change to (200 * 9) for 1.8° stepper
+    #define RA_STEPPER_SPR (400 * 9)  // change to (200 * 9) for 1.8° stepper
 #endif
 #ifndef DEC_STEPPER_SPR
     #define DEC_STEPPER_SPR (200 * 50 * 4.5f)  // change to (200 * 9) for 1.8° stepper
diff --git a/src/EPROMStore.cpp b/src/EPROMStore.cpp
index bb20c8e..b82d7be 100644
--- a/src/EPROMStore.cpp
+++ b/src/EPROMStore.cpp
@@ -469,11 +469,11 @@ void EEPROMStore::storeDECStepsPerDegree(float decStepsPerDegree)
 // If it is not present then the default uncalibrated AZ_STEPS_PER_DEGREE value is returned.
 float EEPROMStore::getAZStepsPerDegree()
 {
-    #if AZ_STEPPER_TYPE != STEPPER_TYPE_NONE
+#if AZ_STEPPER_TYPE != STEPPER_TYPE_NONE
     float azStepsPerDegree(AZIMUTH_STEPS_PER_REV / 360);  // Default value
-    #else
+#else
     float azStepsPerDegree(1);  // Default value
-    #endif
+#endif
 
     if (isPresentExtended(AZ_NORM_STEPS_MARKER_FLAG))
     {
@@ -503,16 +503,15 @@ void EEPROMStore::storeAZStepsPerDegree(float azStepsPerDegree)
     commit();  // Complete the transaction
 }
 
-
 // Return the ALT steps per degree (actually microsteps per degree).
 // If it is not present then the default uncalibrated ALT_STEPS_PER_DEGREE value is returned.
 float EEPROMStore::getALTStepsPerDegree()
 {
-    #if (ALT_STEPPER_TYPE != STEPPER_TYPE_NONE)
+#if (ALT_STEPPER_TYPE != STEPPER_TYPE_NONE)
     float azStepsPerDegree(ALTITUDE_STEPS_PER_REV / 360);  // Default value
-    #else
+#else
     float azStepsPerDegree(1);  // Default value
-    #endif
+#endif
 
     if (isPresentExtended(ALT_NORM_STEPS_MARKER_FLAG))
     {
@@ -542,7 +541,6 @@ void EEPROMStore::storeALTStepsPerDegree(float azStepsPerDegree)
     commit();  // Complete the transaction
 }
 
-
 int16_t EEPROMStore::getLastFlashedVersion()
 {
     if (isPresentExtended(LAST_FLASHED_MARKER_FLAG))
diff --git a/src/MeadeCommandProcessor.cpp b/src/MeadeCommandProcessor.cpp
index 7bbe7ec..0788954 100644
--- a/src/MeadeCommandProcessor.cpp
+++ b/src/MeadeCommandProcessor.cpp
@@ -1858,11 +1858,11 @@ String MeadeCommandProcessor::handleMeadeExtraCommands(String inCmd)
         }
         else if ((inCmd[1] == 'A') && (inCmd.length() == 2))  // :XGA#
         {
-            return String(_mount->getStepsPerDegree(ALTITUDE_STEPS),1) + "#";
+            return String(_mount->getStepsPerDegree(ALTITUDE_STEPS), 1) + "#";
         }
         else if ((inCmd[1] == 'Z') && (inCmd.length() == 2))  // :XGZ#
         {
-            return String(_mount->getStepsPerDegree(AZIMUTH_STEPS),1) + "#";
+            return String(_mount->getStepsPerDegree(AZIMUTH_STEPS), 1) + "#";
         }
         else if ((inCmd[1] == 'A') && (inCmd.length() > 2) && (inCmd[2] == 'H'))  // :XGAH#
         {
diff --git a/src/Mount.cpp b/src/Mount.cpp
index 9ceca38..bd2a94b 100644
--- a/src/Mount.cpp
+++ b/src/Mount.cpp
@@ -78,7 +78,7 @@ Mount::Mount(LcdMenu *lcdMenu)
 
 {
     _commandReceived = 0;
-    _lcdMenu = lcdMenu;
+    _lcdMenu         = lcdMenu;
     initializeVariables();
 }
 
@@ -190,15 +190,15 @@ void Mount::readPersistentData()
     _stepsPerDECDegree = EEPROMStore::getDECStepsPerDegree();
     LOG(DEBUG_INFO, "[MOUNT]: EEPROM: DEC steps/deg is %f", _stepsPerDECDegree);
 
-    #if (AZ_STEPPER_TYPE != STEPPER_TYPE_NONE)
+#if (AZ_STEPPER_TYPE != STEPPER_TYPE_NONE)
     _stepsPerAZDegree = EEPROMStore::getAZStepsPerDegree();
     LOG(DEBUG_INFO, "[MOUNT]: EEPROM: AZ steps/deg is %f", _stepsPerAZDegree);
-    #endif
+#endif
 
-    #if (ALT_STEPPER_TYPE != STEPPER_TYPE_NONE)
+#if (ALT_STEPPER_TYPE != STEPPER_TYPE_NONE)
     _stepsPerALTDegree = EEPROMStore::getALTStepsPerDegree();
     LOG(DEBUG_INFO, "[MOUNT]: EEPROM: ALT steps/deg is %f", _stepsPerALTDegree);
-    #endif
+#endif
     float speed = EEPROMStore::getSpeedFactor();
     LOG(DEBUG_INFO, "[MOUNT]: EEPROM: Speed factor is %f", speed);
     setSpeedCalibration(speed, false);
@@ -1003,19 +1003,19 @@ float Mount::getStepsPerDegree(StepperAxis which)
     }
     if (which == AZIMUTH_STEPS)
     {
-        #if (AZ_STEPPER_TYPE != STEPPER_TYPE_NONE)
+#if (AZ_STEPPER_TYPE != STEPPER_TYPE_NONE)
         return _stepsPerAZDegree;  // u-steps/degree
-        #else
+#else
         return 1;
-        #endif
+#endif
     }
     if (which == ALTITUDE_STEPS)
     {
-        #if (ALT_STEPPER_TYPE != STEPPER_TYPE_NONE)
+#if (ALT_STEPPER_TYPE != STEPPER_TYPE_NONE)
         return _stepsPerALTDegree;  // u-steps/degree
-        #else
+#else
         return 1;
-        #endif
+#endif
     }
 
     return 0;
@@ -1040,20 +1040,20 @@ void Mount::setStepsPerDegree(StepperAxis which, float steps)
         EEPROMStore::storeRAStepsPerDegree(_stepsPerRADegree);
         setSpeedCalibration(_trackingSpeedCalibration, false);
     }
-    #if (AZ_STEPPER_TYPE != STEPPER_TYPE_NONE)
+#if (AZ_STEPPER_TYPE != STEPPER_TYPE_NONE)
     else if (which == AZIMUTH_STEPS)
     {
         _stepsPerAZDegree = steps;
         EEPROMStore::storeAZStepsPerDegree(_stepsPerAZDegree);
     }
-    #endif
-    #if (ALT_STEPPER_TYPE != STEPPER_TYPE_NONE)
+#endif
+#if (ALT_STEPPER_TYPE != STEPPER_TYPE_NONE)
     else if (which == ALTITUDE_STEPS)
     {
         _stepsPerALTDegree = steps;
         EEPROMStore::storeALTStepsPerDegree(_stepsPerALTDegree);
     }
-    #endif
+#endif
 }
 
 /////////////////////////////////
@@ -1621,13 +1621,13 @@ void Mount::stopGuiding(bool ra, bool dec)
         // Stop DEC guiding and wait for it to stop.
         _stepperGUIDE->stop();
 
-    #if !defined(ESP32BOARD)
+#if !defined(ESP32BOARD)
         while (_stepperGUIDE->isRunning())
         {
             _stepperGUIDE->run();
             _stepperTRK->runSpeed();
         }
-    #endif
+#endif
 
         _mountStatus &= ~STATUS_GUIDE_PULSE_DEC;
     }
@@ -2678,9 +2678,7 @@ void Mount::waitUntilStopped(byte direction)
 // Block until all steppers are stopped
 void Mount::waitUntilAllStopped()
 {
-    while (_stepperRA->isRunning()
-           || _stepperDEC->isRunning()
-           || (((_mountStatus & STATUS_TRACKING) == 0) && _stepperTRK->isRunning())
+    while (_stepperRA->isRunning() || _stepperDEC->isRunning() || (((_mountStatus & STATUS_TRACKING) == 0) && _stepperTRK->isRunning())
 #if FOCUS_STEPPER_TYPE != STEPPER_TYPE_NONE
            || _stepperFocus->isRunning()
 #endif
@@ -3028,12 +3026,12 @@ void Mount::loop()
         // One of the motors was running last time through the loop, but not anymore, so shutdown the outputs.
         disableAzAltMotors();
         _azAltWasRunning = false;
-        #if (AZ_STEPPER_TYPE != STEPPER_TYPE_NONE)
-            EEPROMStore::storeAZPosition(_stepperAZ->currentPosition());
-        #endif
-        #if (ALT_STEPPER_TYPE != STEPPER_TYPE_NONE)
-            EEPROMStore::storeALTPosition(_stepperALT->currentPosition());
-        #endif
+    #if (AZ_STEPPER_TYPE != STEPPER_TYPE_NONE)
+        EEPROMStore::storeAZPosition(_stepperAZ->currentPosition());
+    #endif
+    #if (ALT_STEPPER_TYPE != STEPPER_TYPE_NONE)
+        EEPROMStore::storeALTPosition(_stepperALT->currentPosition());
+    #endif
     }
 
     oneIsRunning = false;
@@ -3305,16 +3303,16 @@ void Mount::updateInfoDisplay()
 {
     #if (INFO_DISPLAY_TYPE != INFO_DISPLAY_TYPE_NONE)
     // If we update this display too often while slewing, the serial port is unable to process commands fast enough. Which makes the driver
-    // timeout, causing ASCOM errors. 
+    // timeout, causing ASCOM errors.
     // We will update at 30Hz when idle, 5Hz when slewing one axis and skip updates when slewing both.
     int refreshRateHz = 30;
-    long now = millis();
-    if ((slewStatus() & (SLEWING_DEC | SLEWING_RA)) == (SLEWING_DEC | SLEWING_RA)) 
+    long now          = millis();
+    if ((slewStatus() & (SLEWING_DEC | SLEWING_RA)) == (SLEWING_DEC | SLEWING_RA))
     {
         return;
     }
-    
-    if (isSlewingRAorDEC()) 
+
+    if (isSlewingRAorDEC())
     {
         refreshRateHz = 5;
     }
diff --git a/src/Mount.hpp b/src/Mount.hpp
index 1d170f9..7cf00a7 100644
--- a/src/Mount.hpp
+++ b/src/Mount.hpp
@@ -406,7 +406,7 @@ class Mount
     void setupInfoDisplay();
     void updateInfoDisplay();
     InfoDisplayRender *getInfoDisplay();
-    long _lastInfoUpdate = 0; // Last time the info display was updated
+    long _lastInfoUpdate = 0;  // Last time the info display was updated
 #endif
 
     // Called by Meade processor every time a command is received.
diff --git a/src/b_setup.hpp b/src/b_setup.hpp
index 2d9e269..5ac294a 100644
--- a/src/b_setup.hpp
+++ b/src/b_setup.hpp
@@ -181,9 +181,9 @@ void setup()
 
     #ifdef RA_SERIAL_PORT
         #ifdef OAE
-        RA_SERIAL_PORT.begin(57600, SERIAL_8N1, RA_TX_PIN, RA_RX_PIN);
+    RA_SERIAL_PORT.begin(57600, SERIAL_8N1, RA_TX_PIN, RA_RX_PIN);
         #else
-        RA_SERIAL_PORT.begin(57600);  // Start HardwareSerial comms with driver
+    RA_SERIAL_PORT.begin(57600);   // Start HardwareSerial comms with driver
         #endif
     //
     #endif
@@ -204,13 +204,13 @@ void setup()
 #if DEC_DRIVER_TYPE == DRIVER_TYPE_TMC2209_UART
     // include TMC2209 UART pins
     #if defined(DEC_DIAG_PIN)
-        pinMode(DEC_DIAG_PIN, INPUT);
+    pinMode(DEC_DIAG_PIN, INPUT);
     #endif
     #ifdef DEC_SERIAL_PORT
         #ifdef OAE
-        DEC_SERIAL_PORT.begin(57600, SERIAL_8N1, DEC_TX_PIN, DEC_RX_PIN);
+    DEC_SERIAL_PORT.begin(57600, SERIAL_8N1, DEC_TX_PIN, DEC_RX_PIN);
         #else
-        DEC_SERIAL_PORT.begin(57600);  // Start HardwareSerial comms with driver
+    DEC_SERIAL_PORT.begin(57600);  // Start HardwareSerial comms with driver
         #endif
     #endif
 #endif
@@ -456,7 +456,7 @@ void setup()
     LOG(DEBUG_ANY, "[STEPPERS]: ALT Circumference : %f", ALT_CIRCUMFERENCE);
     LOG(DEBUG_ANY, "[STEPPERS]: ALT steps/rev     : %f", ALTITUDE_STEPS_PER_REV);
     LOG(DEBUG_ANY, "[STEPPERS]: ALT steps/deg     : %f", mount.getStepsPerDegree(ALTITUDE_STEPS));
-    LOG(DEBUG_ANY, "[STEPPERS]: ALT steps/minute  : %f", ALTITUDE_STEPS_PER_ARC_MINUTE);    
+    LOG(DEBUG_ANY, "[STEPPERS]: ALT steps/minute  : %f", ALTITUDE_STEPS_PER_ARC_MINUTE);
     mount.configureALTStepper(ALTmotorPin1, ALTmotorPin2, ALT_STEPPER_SPEED, ALT_STEPPER_ACCELERATION);
     #if ALT_DRIVER_TYPE == DRIVER_TYPE_TMC2209_UART
     LOG(DEBUG_ANY, "[STEPPERS]: Configure ALT driver...");
diff --git a/src/testmenu.cpp b/src/testmenu.cpp
index 0dcabb5..b046bf4 100644
--- a/src/testmenu.cpp
+++ b/src/testmenu.cpp
@@ -261,12 +261,12 @@ void TestMenu::listHardware() const
     int index  = 0;
     Serial.print(F("              Mount: "));
     #ifdef OAM
-        Serial.println(F("OpenAstroMount (OAM)"));
+    Serial.println(F("OpenAstroMount (OAM)"));
     #else
         #ifdef OAE
-            Serial.println(F("OpenAstroExplorer (OAE)"));
+    Serial.println(F("OpenAstroExplorer (OAE)"));
         #else
-            Serial.println(F("OpenAstroTracker (OAT)"));
+    Serial.println(F("OpenAstroTracker (OAT)"));
         #endif
     #endif
 
@@ -534,9 +534,9 @@ void TestMenu::display() const
         Serial.println(F("*** OpenAstroMount (OAM) Test Menu ***"));
     #else
         #ifdef OAE
-            Serial.println(F("*** OpenAstroTracker (OAE) Test Menu ***"));
+        Serial.println(F("*** OpenAstroTracker (OAE) Test Menu ***"));
         #else
-            Serial.println(F("** OpenAstroTracker (OAT) Test Menu **"));
+        Serial.println(F("** OpenAstroTracker (OAT) Test Menu **"));
         #endif
     #endif
         Serial.print(F("************* "));
